### DISCUSSION TITLE
FIBEX File Generation out of DLT - Parser

### DISCUSSION THREAD

**Discussion by girishchidanand** [COVESA/dlt-viewer#275] [General]
Hello!!!

I would like to generate a fibex file out of DLT-Parser, Any help with documentation or guidance on how to achieve this would be much appreciated!!!

**Comment by alexmucde** [comment]
@girishchidanand Please consider the AUTOSAR DLT standard and all examples in DLT Parser. There is no extra documentation available beside these.
